MyScript/index.js

```diff
@@ -0,0 +1,67 @@
+//const inputDate = '11/27/2021'
+const inputDate = prompt('Enter the date You are interested in: ', '11/26/2021')
+const inputDateFull = new Date(inputDate)
+
+const checkEnteredDay = (enteredDate) => {
+    const twoDayMs = 86400000 * 2
+    const firstWorkDayMs = new Date(2021, 10, 26).getTime()
+
+    const inputDateMs = new Date(enteredDate).getTime()
+    const differenceMs = inputDateMs - firstWorkDayMs
+    const divisionResult = differenceMs / twoDayMs
+    const reminder = divisionResult % twoDayMs
+    const checkSecondDay = Math.round(reminder) % 2
+    const isFirstWorkDay = !(reminder % 2)
+    const isFirstFreeDay = !isFirstWorkDay && Number.isInteger(reminder)
+    const isSecondFreeDay = !isFirstWorkDay && !checkSecondDay
+    const isSecondWorkDay = !isFirstWorkDay && checkSecondDay
+
+    let resultObj = {
+        day: '',
+        busyness: ''
+    }
+
+    if (isFirstWorkDay) {
```

Comment:
All 4 ```if``` blocks perform the same operation, they're basically setting contents for ```resultObj```. These blocks are redundant. It would be better to make separate ```isFirstDay``` and ```isDayFree``` checks, and do like that:

```
const resultObj = {
    day: isFirstDay ? 'first' : 'second'
    busyness: isDayFree ? 'REST' : 'WORK'
}
```